libraries/dagster-evidence/dagster_evidence/__init__.py

```diff
@@ -0,0 +1,5 @@
+from dagster._core.libraries import DagsterLibraryRegistry
+
+__version__ = "0.26.9"
```

Comment:
Nit: This should probably be 0.1.0 or something, since `community-integrations` aren't released weekly/in sync with Dagster.